test/functional/JLM_Tests/src/org/openj9/test/management/TestGetSystemCpuLoad.java

```diff
@@ -38,97 +38,63 @@
-			validateLoad(load, true, CpuLoadCalculationConstants.UNSUPPORTED_VALUE, "initial call");
-			return;
-		} else {
-			validateLoad(load, true, CpuLoadCalculationConstants.ERROR_VALUE, "initial call");
-		}
-		load = ibmBean.getSystemCpuLoad();
-		if (load < 0.0) { /* normal case (insufficient time since last call) */
-			validateLoad(load, true, CpuLoadCalculationConstants.ERROR_VALUE, "call getSystemCpuLoad immediately after the previous call");
-		} else { /* possible but unlikely: test stalled for an extended time */
-			validateLoad(load, false, NO_ERROR, "call getSystemCpuLoad immediately after the previous call");
-		}
-
-		delayMillis(MINIMUM_INTERVAL);
-		load = ibmBean.getSystemCpuLoad();
-		validateLoad(load, false, NO_ERROR, "call getSystemCpuLoad after the minumum interval");
-
-		delayMillis(1000);
-		load = ibmBean.getSystemCpuLoad();
-		validateLoad(load, false, NO_ERROR, "call getSystemCpuLoad 1 second after previous call");
-
-		delayMillis(1);
-		load = ibmBean.getSystemCpuLoad();
-		validateLoad(load, false, NO_ERROR, "call getSystemCpuLoad 1 millisecond after previous call");
 	}
 
 	@Test
 	public void testMxBean() {
 		logger.debug("Starting testMxBean");
 		java.lang.management.OperatingSystemMXBean osBean = ManagementFactory.getOperatingSystemMXBean();
 		String beanClass = osBean.getClass().getName();
-		logger.debug("osBean class=" + beanClass);
+		logger.debug("osBean class = " + beanClass);
 		Class<?> ibmBeanClass = com.ibm.lang.management.OperatingSystemMXBean.class;
 
 		if (ibmBeanClass.isInstance(osBean)) {
-			com.ibm.lang.management.OperatingSystemMXBean ibmBean1 =
-					(com.ibm.lang.management.OperatingSystemMXBean) osBean;
-			double load = ibmBean1.getSystemCpuLoad();
-			if (!supported) {
-				validateLoad(load, true, CpuLoadCalculationConstants.UNSUPPORTED_VALUE, "initial getSystemCpuLoad");
-				return;
-			} else if (load < 0.0) { /* normal case (insufficient time since last call) */
-				validateLoad(load, true, CpuLoadCalculationConstants.ERROR_VALUE, "initial getSystemCpuLoad");
-			} else {
-				validateLoad(load, false, NO_ERROR, "initial getSystemCpuLoad");
-			}
-			delayMillis(MINIMUM_INTERVAL);
-			load = ibmBean1.getSystemCpuLoad();
-			validateLoad(load, false, NO_ERROR, "getSystemCpuLoad after the minumum interval");
-
-			load = ibmBean1.getSystemCpuLoad();
-			validateLoad(load, false, NO_ERROR, "getSystemCpuLoad again");
-
-			delayMillis(MINIMUM_INTERVAL);
-			load = ibmBean1.getSystemCpuLoad();
-			validateLoad(load, false, NO_ERROR, "getSystemCpuLoad again");
-
-			delayMillis(1000);
-			load = ibmBean1.getSystemCpuLoad();
-			validateLoad(load, false, NO_ERROR, "getSystemCpuLoad after 1 second");
+			testMxBeanImpl((com.ibm.lang.management.OperatingSystemMXBean)osBean);
 		} else {
 			Assert.fail("OperatingSystemMXBean is wrong type: " + osBean.getClass().getName());
 		}
 	}
 
+	public static void testMxBeanImpl(com.ibm.lang.management.OperatingSystemMXBean ibmBean) {
```

Comment:
Why is this `public` and not `private`?

Reply:
Fixed in [force-push](https://github.com/eclipse/openj9/compare/e200fcd0089727bb8c11ffa2d51bc48880865951..a15fd0a5f1583c638085002e198a4ef9732245cf).